? addressbook/libebook/libebook-1.2.pc
? addressbook/libebook/libebook.pc
? addressbook/libedata-book/libedata-book-1.2.pc
? addressbook/libedata-book/libedata-book.pc
? calendar/backends/weather
? calendar/libecal/libecal-1.2.pc
? calendar/libecal/libecal.pc
? calendar/libedata-cal/libedata-cal-1.2.pc
? calendar/libedata-cal/libedata-cal.pc
? calendar/libical/src/libicalss/icalssyacc.output
? libedataserver/libedataserver-1.2.pc
? libedataserver/libedataserver.pc
? servers/groupwise/libegroupwise.pc
? src/GNOME_Evolution_DataServer_1.2.server
Index: configure.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/configure.in,v
retrieving revision 1.70
diff -u -r1.70 configure.in
--- configure.in	30 Sep 2004 16:47:45 -0000	1.70
+++ configure.in	2 Oct 2004 07:55:23 -0000
@@ -318,6 +318,7 @@
 calendar/backends/groupwise/Makefile
 calendar/backends/http/Makefile
 calendar/backends/contacts/Makefile
+calendar/backends/weather/Makefile
 calendar/tests/Makefile
 calendar/tests/ecal/Makefile
 libdb/Makefile
Index: calendar/backends/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/backends/Makefile.am,v
retrieving revision 1.3
diff -u -r1.3 Makefile.am
--- calendar/backends/Makefile.am	9 Jan 2004 20:05:27 -0000	1.3
+++ calendar/backends/Makefile.am	2 Oct 2004 07:55:24 -0000
@@ -1 +1 @@
-SUBDIRS = file groupwise http contacts
+SUBDIRS = file groupwise http contacts weather
Index: calendar/libical/src/libicalss/icalssyacc.h
===================================================================
RCS file: /cvs/gnome/libical/src/libicalss/icalssyacc.h,v
retrieving revision 1.12
diff -u -r1.12 icalssyacc.h
--- calendar/libical/src/libicalss/icalssyacc.h	28 May 2004 10:14:24 -0000	1.12
+++ calendar/libical/src/libicalss/icalssyacc.h	2 Oct 2004 07:55:24 -0000
@@ -78,7 +78,7 @@
 typedef union YYSTYPE {
 	char* v_string;
 } YYSTYPE;
-/* Line 1249 of yacc.c.  */
+/* Line 1240 of yacc.c.  */
 #line 82 "icalssyacc.h"
 # define yystype YYSTYPE /* obsolescent; will be withdrawn */
 # define YYSTYPE_IS_DECLARED 1
